lib/web/app/transport.go

```diff
@@ -193,12 +193,15 @@ func (t *transport) rewriteRedirect(resp *http.Response) error {
 	// We want the rewrite to happen using our own public address.
 	if host == t.c.identity.RouteToApp.PublicAddr {
 		// drop scheme and host, leaving only the relative path.
-		// since the path can be an empty string, canonicalize it as "/".
+		u.Host = ""
+		u.Scheme = ""
```

Comment:
So the comment was right after all? :sweat_smile:

Reply:
Well, the comment was right, only the implementation had issues 😄 not with the host and scheme though, that worked just fine.